third-party-programs.txt

```diff
@@ -312,9 +312,201 @@ INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT
 STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY
 WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
--------------------------------------------------------------
+=====================================================================================
 
 4. CuML
+   Copyright 2018 NVIDIA CORPORATION
+   
+   xgboost
```

Comment:
I'm confused. Is `xgboost` to be understood as part of `CuML`? Or should it be

```
5. xgboost
```

In this case, what's the `CuML` license? Latest branch mentions Apache 2.0
https://github.com/rapidsai/cuml/blob/branch-23.12/LICENSE

Reply:
That file comes from legal. Point #4 only groups the components that obtain the same license in order to save space in the file. In the case of XGBoost and cuML it's Apache 2.0.